crates/transaction-emitter-lib/src/args.rs

```diff
@@ -164,6 +164,9 @@ pub struct EmitArgs {
     #[clap(long)]
     pub max_gas_per_txn: Option<u64>,
 
+    #[clap(long)]
+    pub init_max_gas_per_txn: Option<u64>,
```

Comment:
What's the difference between `max_gas_per_txn` and `init_max_gas_per_txn`?

Reply:
in transaction generator, there are set of transactions we use to initialize emitter, that we don't care about throughput (like creating worker accounts, publishing modules etc), but that need to go through - no matter what

those are called "init" transactions, and we can set different gas price, and now different max_gas